openapi_python_client/parser/openapi.py

```diff
@@ -265,6 +265,15 @@ def from_data(
 
         return result, schemas
 
+    def response_type(self) -> str:
```

Comment:
Any reason this isn't a `@property`?

Reply:
Generally bad practice to make something a property if it does a bunch of work. The method call `()` indicates to the caller that something is happening, so if you need it twice you should assign it to a variable rather than calling it twice.